modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,116 @@
+= Business Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Business groups provide a way to separate and control access to Anypoint Platform resources such as APIs and applications. Users have access only to the business groups to which they are added.  
+
+Business groups reside within the master organization and can contain their own child business groups. The master organization and its business groups are organized in a hierarchical tree, where the master organization is the root. Each business group has one direct parent and can have multiple children.
+
+[NOTE]
+Business groups are not enabled by default in a new Anypoint Platform account. To activate business groups in your organization, contact your MuleSoft representative. There is a limit of 100 business groups per organization.
+
+If you are an organization administrator and your organization is enabled for business group support, you can create business groups within your organization. The organization administrator is also the administrator of any business groups (sub-organizations) within the parent organization.
+
+[NOTE]
+There is a limit of 100 business groups per organization. 
+
+==== Child Business Groups
+
+Child business groups provide more fine grained control over access. 
+
+image::organization-2b25c.png[]
+
+Since business groups are hierarchical, the owner of a parent business group always retains administrator permissions for any child business group within that parent. This means that the owner of a business group can create child business groups and assign another user as the owner (owners automatically have the *Organization Administrators* role) of those child business groups. The owner of the parent group remains an *Organization Administrator* within the child business group. 
```

Comment:
_Always_ isn't true. They always retain the ability to make themselves owners and admins, but the hierarchy grants admin access at BG creation time. Therefore, if a user is added as an admin of the parent org later, they won't have automatic admin of the BG